src/tree/SubscriptionTreeItem.ts

```diff
@@ -46,7 +46,7 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         const treeItemPostgres: AzExtTreeItem[] = await this.createTreeItemsWithErrorHandling(
             postgresServers,
             'invalidPostgreSQLAccount',
-            async (server: PostgresAbstractServer) => await this.initPostgresChild(server),
+            async (server: PostgresAbstractServer) => await this.initPostgresChild(server, context),
```

Comment:
The problem with doing it in SubscriptionTreeItem is that each child will overwrite the previous child's value. I would suggest you add this telemetry in PostgresServerTreeItem.loadMoreChildrenImpl instead. If you want to do it here, maybe add two separate properties `hasFlexibleServer` and `hasSingleServer` or something like that